### DISCUSSION TITLE
Display with vim keys

### DISCUSSION THREAD

**Discussion by yougotwill** [rxhanson/Charmstone-Community#51] [Ideas]
This is an awesome tool and is a great alternative for moving around spaces since it is now broken in most window managers!

I use a vim layout for most of my work and it would be great if we could use vim keys `h j k l` as an alternative to arrow keys.

Keep up the great work 💪

**Comment by rxhanson** [comment]
Thanks for the feedback! This one's on my backlog, but I haven't made it there yet. Hopefully soon!
